vars/waitForHelixRuns.groovy

```diff
@@ -127,16 +141,12 @@ def call (def helixRunsBlob, String prStatusPrefix) {
 
                 if (isPending && state == 0) {
                     state = 1
-                    setPRStatus(context, "PENDING", "", "Waiting")
                 }
                 else if (isRunning) {
                     state = 2
-                    setPRStatus(context, "PENDING", mcResultsUrl, subMessage)
                 }
                 else if (isFinished) {
                     state = 3
-
-                    setPRStatus(context, resultValue, mcResultsUrl, subMessage)
```

Comment:
One critical thing here is that we want to fail the build if the tests fail, otherwise it will just always show green in the GitHub UI.

Here, the waitUntil step is a task that just waits until the closure returns true:
`waitUntil { if (done) { return true } else { return false } }`

We want to propagate the result of the helix job outside of the waitUntil and into the main parallel statement.  I believe it's something like:
`waitUntil {
 // Check Helix status
 if (done) { passed = ???; return true }
}`
if (!passed) { error "Please check foo" }